docs/Users/Install.rst

```diff
@@ -153,6 +153,11 @@ If you want to install coala to an alternate location you can e.g. call
 options are documented on
 https://docs.python.org/3.3/install/#alternate-installation.
 
+..note::
+
+    Make sure you do not use any proxy at installation
```

Comment:
I've set this up behind a proxy, I think setting the pip proxy or exporting http_proxy and https_proxy should solve the problem. I don't think putting a note not to install behind a proxy is the solution to that.